Dockerfile

```diff
@@ -22,6 +22,12 @@ FROM node AS gulp
 
 WORKDIR /app
 
+RUN apt-get update && apt-get install --yes --no-install-recommends \
+        python-pip \
```

Comment:
Unfortunately have to delegate to Python; `pyftsubset` seems to be the standard.

Reply:
Python 2 is also unfortunate, but had problems with Python 3. (Though have changed things since, so will have another go.)

Reply:
🎉 for 3!